recipes/fruit/all/conanfile.py

```diff
@@ -54,69 +58,56 @@ def configure(self):
            compiler_version < minimal_version[compiler]:
-            raise ConanInvalidConfiguration("%s requires a compiler that supports"
-                                            " at least C++11. %s %s is not"
-                                            " supported." % (self.name, compiler, compiler_version))
+            raise ConanInvalidConfiguration(f"{self.name} requires a compiler that supports"
+                                            " at least C++11. {compiler} {compiler_version} is not"
+                                            " supported.")
 
-        if self.settings.compiler.cppstd:
-            tools.check_min_cppstd(self, "11")
-
-    @property
-    def _extracted_dir(self):
-        return self.name + "-" + self.version
-
-    def _get_source(self):
+    def source(self):
         if Version(self.version) == "3.4.0":
             filename = os.path.basename(self.conan_data["sources"][self.version]["url"])
-            tools.download(filename=filename, **self.conan_data["sources"][self.version])
+            download(self, filename=filename, **self.conan_data["sources"][self.version])
+            extracted_dir = self.name + "-" + self.version
 
             with tarfile.TarFile.open(filename, 'r:*') as tarredgzippedFile:
                 # NOTE: In fruit v3.4.0, The archive file contains the file names
                 # build and BUILD in the extras/bazel_root/third_party/fruit directory.
                 # Extraction fails on a case-insensitive file system due to file
                 # name conflicts.
                 # Exclude build as a workaround.
-                exclude_pattern = "%s/extras/bazel_root/third_party/fruit/build" % (self._extracted_dir,)
+                exclude_pattern = f"{extracted_dir}/extras/bazel_root/third_party/fruit/build"
                 members = list(filter(lambda m: not fnmatch(m.name, exclude_pattern),
                                     tarredgzippedFile.getmembers()))
-                tarredgzippedFile.extractall(".", members=members)
+                tarredgzippedFile.extractall(path=self.source_folder, members=members)
+            allfiles = os.listdir(os.path.join(self.source_folder, extracted_dir))
+            for file_name in allfiles:
+                shutil.move(os.path.join(self.source_folder, extracted_dir, file_name), self.source_folder)
         else:
-            tools.get(**self.conan_data["sources"][self.version])
-
-    def source(self):
-        self._get_source()
-
-        os.rename(self._extracted_dir, self._source_subfolder)
-
-    def _configure_cmake(self):
-        if not self._cmake:
-            self._cmake = CMake(self)
-            self._cmake.definitions["FRUIT_USES_BOOST"] = self.options.use_boost
-            self._cmake.definitions["FRUIT_ENABLE_COVERAGE"] = False
-            self._cmake.definitions["RUN_TESTS_UNDER_VALGRIND"] = False
-            self._cmake.definitions["FRUIT_ENABLE_CLANG_TIDY"] = False
```

Comment:
Sorry if this is a dumb question, but can you tell me why this definition was left out of the v2 conversion? I'm guessing it does not make sense to have it anymore? Or is it releated to https://github.com/conan-io/conan-center-index/pull/15494/files#diff-9a0d263d27f331b7f71ed0a2f4e39864d4e1ba6df602130a0094d84b52109612R434?

Reply:
It's False already by default: https://github.com/google/fruit/blob/v3.7.0/CMakeLists.txt